std/math/floats.d

```diff
@@ -0,0 +1,16 @@
+// Written in the D programming language.
+
+/**
+This package is currently in a nascent state and may be subject to
+change. Please do not use it yet, but stick to $(MREF std, math).
+
+Copyright: Copyright The D Language Foundation 2000 - 2011.
+License:   $(HTTP www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+Authors:   $(HTTP digitalmars.com, Walter Bright), Don Clugston,
+           Conversion of CEPHES math library to D by Iain Buclaw and David Nadlinger
+Source: $(PHOBOSSRC std/math/floats.d)
+ */
+
+module std.math.floats;
```

Comment:
Don't do anything yet, but I wonder if this could be broken down into two separate modules as well i.e: `std.math.comparison` for `feqrel`, `isClose`, ... and `std.math.floatmanip` (for want of a better name) for `nextDown`, `ilogb`, ...

Reply:
Sounds like a good idea. I'll wait for other comments on this.

Reply:
An other approach might be to put the whole floating point stuff (this,  the hardware stuff, and maybe even `CustomFloat` from `std.numeric`) into a separate package, for example `std.floats`, with own submodules. In the end, the focus of this stuff is not on math, but on floating points.

Reply:
Wrote a [forum post](https://forum.dlang.org/post/atovzoedfzlvujfvpvkb@forum.dlang.org) asking this. Let's see what happens.